icu4c/source/data/makedata.mak

```diff
@@ -284,7 +284,13 @@ ICU_DATA_BUILD_VERBOSE=
 # Three main targets: tools, core data, and test data.
 # Keep track of whether they are built via timestamp files.
 
-$(TOOLS_TS): "$(ICUTOOLS)\genrb\$(CFGTOOLS)\genrb.exe" "$(ICUTOOLS)\gencnval\$(CFGTOOLS)\gencnval.exe" "$(ICUTOOLS)\gencfu\$(CFGTOOLS)\gencfu.exe" "$(ICUTOOLS)\icupkg\$(CFGTOOLS)\icupkg.exe" "$(ICUTOOLS)\makeconv\$(CFGTOOLS)\makeconv.exe" "$(ICUPBIN)\pkgdata.exe"
+GENRB_DIR=$(ICUTOOLS)\genrb\$(CFGTOOLS)
+GENCNVAL_DIR=$(ICUTOOLS)\gencnval\$(CFGTOOLS)
+GENCFU_DIR=$(ICUTOOLS)\gencfu\$(CFGTOOLS)
+ICUPKG_DIR=$(ICUTOOLS)\icupkg\$(CFGTOOLS)
+MAKECONV_DIR=$(ICUTOOLS)\makeconv\$(CFGTOOLS)
+
+$(TOOLS_TS): "$(GENRB_DIR)\genrb.exe" "$(GENCNVAL_DIR)\gencnval.exe" "$(GENCFU_DIR)\gencfu.exe" "$(ICUPKG_DIR)\icupkg.exe" "$(MAKECONV_DIR)\makeconv.exe" "$(ICUPBIN)\pkgdata.exe"
```

Comment:
flattened out the build directories?

Reply:
Yeah these are some "hacks" with which I wanted to get the real, non-stub, icudt dll to generate. I'll look into cleaning up and making this PR actually functioning tomorrow.